plugins/aws/s3/src/hopeit/aws/s3/object_storage.py

```diff
@@ -436,6 +390,46 @@ def partition_key(self, path: str) -> str:
             partition_key = path.rsplit("/", 1)[0]
         return partition_key
 
+    async def create_bucket(self, check_if_exists: bool = True) -> bool:
```

Comment:
implement using signature from os.makedirs https://docs.python.org/3/library/os.html#os.makedirs

Reply:
address in the last commit!